src/nifake/tests/test_session.py

```diff
@@ -328,3 +328,34 @@ def test_multiple_return_params(self):
             assert self.patched_errors.handle_error.call_count == 2
             self.patched_errors.handle_error.assert_called_with(session, self.patched_library.niFake_GetCalDateAndTime.return_value)
     '''
+
+    def test_init_with_options_nondefault(self):
+        session = nifake.Session('FakeDevice', 1, True, 'Some string')
+        assert(session.vi == SESSION_NUM_FOR_TEST)
+        self.patched_library.niFake_InitWithOptions.assert_called_once_with(b'FakeDevice', 1, True, ANY, ANY)
```

Comment:
Don't use ANY, use the parameters values you expect. Otherwise we aren't ensuring the DLL gets called correctly.